homeassistant/components/opengarage/cover.py

```diff
@@ -112,11 +112,9 @@ def extra_state_attributes(self):
         return self._extra_state_attributes
 
     @property
```

Comment:
We shouldn't implement `state` directly as the platform wants `is_closed`, `is_opening`, and `is_closing`

https://developers.home-assistant.io/docs/core/entity/cover/#platform-properties-to-be-implemented-by-deriving-platform-classes